info.log

```diff
@@ -0,0 +1,594 @@
+2021-09-20T18:35:15.515Z info: [main df2104d] ⭕ Create incidents.yml
```

Comment:
*.log files shouldn't be committed. May delete *.log files in an upcoming commit.